src/Sqids/SqidsEncoder.cs

```diff
@@ -12,23 +28,56 @@ public sealed class SqidsEncoder
 	private readonly int _minLength;
 	private readonly string[] _blockList;
 
+#if NET7_0_OR_GREATER
+	/// <summary>
+	/// The minimum numeric value that can be encoded/decoded using <see cref="SqidsEncoder{T}" />.
+	/// This is always zero across all ports of Sqids.
+	/// </summary>
+	public static T MinValue => T.Zero;
+#else
 	/// <summary>
 	/// The minimum numeric value that can be encoded/decoded using <see cref="SqidsEncoder" />.
 	/// This is always zero across all ports of Sqids.
 	/// </summary>
 	public const int MinValue = 0;
+#endif
 
+#if NET7_0_OR_GREATER
+	/// <summary>
+	/// The maximum numeric value that can be encoded/decoded using <see cref="SqidsEncoder{T}" />.
+	/// It's equal to `int.MaxValue`.
```

Comment:
This isn't `int.MaxValue` anymore. It should be `T.MaxValue` or the equivalent xmldoc.

Reply:
Good catch!